Earthfile

```diff
@@ -52,16 +52,9 @@ build-final-spec:
     SAVE ARTIFACT build/latest.json
 
 openapi:
-    FROM core+base-image
-    COPY . /src
```

Comment:
please avoid that.
COPY . will copy the entire repository remotely.
And the SAVE ARTIFACT will copy the entire repository again.